roles/provision_vcenter/README.md

```diff
@@ -81,6 +81,9 @@ pyvomi
 - **provision_vcenter_vm_network_dns**:
   - list(str), A list of DNS servers the vcenter vm should use. Required if network mode is set to 'static'.
 
+- **provision_vcenter_vm_ntp_server**:
+  - str or int, Server to use for NTP source.
```

Comment:
is `int` for IPs, or is there another way to specify the NTP servers besides hostname/IP?

Reply:
Yes just for pure IPs if used. I was just copying what was done for the other IP vars. It probably should only be str.